### ISSUE TITLE
Cannot pin flatpak to KDE Plasma icon-only task manager/taskbar. Missing metadata?

### ISSUE THREAD

**Issue by ghtesting2020** [flathub/org.videolan.VLC#141]
I am unable to pin the VLC flatpak icon to the KDE Plasma icon-only task manager (the bar at the bottom) also called taskbar on Windows.

I have never experienced this before with any other flatpaks where I cannot "Pin to Task Manager" when I right click the icon in KDE Plasma. The option is greyed out for the VLC icon but all of my other flatpaks pin without issue.

It is a pain having to use the application launcher every single time I want to open VLC the next day. I did find a way to "pin it" by right clicking in the application launcher (start menu for Windows users) and "Pin to taskbar" but that only creates a shortcut in the task manager when you click it that actually launches another instance of VLC so now there would be 2 VLC icons in the task manager. Ideally I would be able to pin it like all my other flatpak apps with right click on the icon and "Pin to task manager".

I did some digging and it seems some users have a similar issue with other flatpaks and the fix is adding meta data or something so the icon-only Plasma knows how to pin the flatpak app icon linked to the flatpak app.

https://bugs.kde.org/show_bug.cgi?id=436465

https://github.com/pkulchenko/ZeroBraneStudio/issues/1114

https://github.com/vector-im/element-web/issues/14619

It is not permission related in flatseal. I have tried enabling and disabling everything.

@MatMaul I noticed this is a similar issue to [123](https://github.com/flathub/org.videolan.VLC/issues/123) so something is definitely up. The 123 ticket is for Cinnamon and this is KDE Plasma. I believe it is metadata that is missing as seen in the links above.

Thank you! VLC is an amazing software.

**Comment by MatMaul** [comment]
I still don't understand why this is happening sorry. as asked in #123, is it possible that you pinned it after opening a DVD ? Does it also happen if you pin directly from the menubar if possible, or after opening it from there ?

**Comment by ghtesting2020** [comment]
> I still don't understand why this is happening sorry. As asked in #123, is it possible that you pinned it after opening a DVD ? Does it also happen if you pin directly from the menubar if possible, or after opening it from there ?

Thanks for the reply. No I am not the same user from #123, I was just mentioning it since I saw you were looking into similar.
1) no I tried pinning right away after installing the flatpak. This was a fresh VLC install on a fresh Linux install
2) I am not sure if menubar you mean something else, but with KDE Plasma I cannot pin after opening VLC and then right clicking on the VLC icon in the Task Manager (in Windows it is called Taskbar) the bar at the bottom. The "Pin to Task Manager" is  greyed out
